README.md

```diff
@@ -7,43 +7,59 @@ Introduction
 ------------
 
 Open Enclave (OE) is an SDK for building enclave applications in C and C++. An
-enclave application partitions itself into two components (1) An untrusted component (called the
-host) and (2) A trusted component (called the enclave). An enclave is a secure
-container whose memory is protected from entities outside the enclave. These
-protections allow enclaves to perform secure computations with assurances that
-secrets will not be compromised.
+enclave application partitions itself into two components:
+1. An untrusted component (called the host) and
+2. A trusted component (called the enclave).
 
-This SDK is a fully open-source and transparent project, which plans to generalize enclave application models across
-enclave implementations from different hardware vendors. It's a non-vendor specific solution that supports enclave applications both on
-Linux and Windows platforms.
+An enclave is a secure container whose memory is protected from entities outside
```

Comment:
There was feedback from some folks in the past that "secure container" was potentially confusing (Anita & Dmitry, I think).

> An _enclave_ is a protected memory region that provides confidentiality for the data and code executing inside it.